src/app/components/users-dashboard/users-dashboard.component.scss

```diff
@@ -0,0 +1,73 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
```

Comment:
Can this file be structured according to the actual template layout? Check the login component css.